views/chitter/index.erb

```diff
@@ -1,8 +1,17 @@
 <!DOCTYPE html>
 <html>
   <head>
-    <meta charset="utf-8">
-    <title></title>
+    <style>
```

Comment:
To keep your view files a bit cleaner, you can create a seperate CSS file and at a link to it in the html. Dom sent me this for the RPS game last week which helped a bit: https://github.com/webdevjeffus/css-for-sinatra#putting-it-in-place